frontend/src/components/Instruction.jsx

```diff
@@ -54,27 +53,30 @@ function Instruction() {
         </li>
       </ul>
       <div className="userButton">
-        <button
-          onClick={() => navigate("/category/hebergement")}
+        <Link
+          to="/categories/:categoryId"
```

Comment:
you must provide a value here for :categoryId. from the previous code, I guess you want to write:

    to="categories/hebergement"